src/coreclr/vm/codeman.cpp

```diff
@@ -3714,11 +3719,15 @@ BOOL EEJitManager::JitCodeToMethodInfo(
 
     TADDR start = dac_cast<PTR_EEJitManager>(pRangeSection->pjit)->FindMethodCode(pRangeSection, currentPC);
     if (start == NULL)
+    {
         return FALSE;
+    }
 
     CodeHeader * pCHdr = PTR_CodeHeader(start - sizeof(CodeHeader));
     if (pCHdr->IsStubCodeBlock())
+    {
```

Comment:
You don't actually need this change. There are other blocks like this all over the place.